### PR TITLE
Add identify controller structures (#3)

### PR DESCRIPTION
**Pull request by csm10495**
Merge from branch.  Just getting structures ready.

### CODE REVIEW

File: cNVMe/Controller.cpp [hunk 1]
```diff
@@ -189,11 +189,7 @@ namespace cnvme
 			COMPLETION_QUEUE_ENTRY completionQueueEntryToPost = { 0 };
 			UINT_32 memoryPageSize = ControllerRegisters->getMemoryPageSize();
 
-			if (memoryPageSize == 0)
```
Comment (csm10495) [inline cNVMe/Controller.cpp:192]: I want to assert here if this fails... may as well die. We can go over cNVMe's asserting if anyone has any questions.

Comment (csm10495) [reply cNVMe/Controller.cpp:192]: (not directly related)

File: cNVMe/Identify.h [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+#include "Types.h"
+
+#pragma once
+
+namespace cnvme
+{
+	namespace identify
+	{
+		namespace structures
```
Comment (csm10495) [inline cNVMe/Identify.h:34]: Is it worth doing another namespace here? (Or is this too many already?)

Comment (mfamis) [reply cNVMe/Identify.h]: I think this is a sufficient number of namespaces (and its plenty descriptive as is).

Comment (mfamis) [reply cNVMe/Identify.h:34]: I think this is a sufficient number of namespaces (and its plenty descriptive as is).

**Pull request merged**